Library/Homebrew/cask/lib/hbc/cli/doctor.rb

```diff
@@ -27,6 +27,7 @@ def run
         check_taps
         check_load_path
         check_environment_variables
+        check_quarantine_support
```

Comment:
Maybe move this into `check_software_versions` so it is listed with `macOS` and `SIP` rather than at the end?

Reply:
This adds an error message if there are missing tools; I can move it up the chain instead, if you want.